pkg/pillar/cmd/zedagent/handlemetrics.go

```diff
@@ -1165,6 +1165,17 @@ func PublishAppInfoToZedCloud(ctx *zedagentContext, uuid string,
 			snapInfo.SnapErr = encodeErrorInfo(snap.Error)
 			ReportAppInfo.Snapshots = append(ReportAppInfo.Snapshots, snapInfo)
 		}
+
+		// For Clustered apps on HV=kubevirt, update controller
+		// with the flag 'ClusterAppRunning' which designates this node
+		// as currently running the app after some failover between nodes.
+		dsItems := ctx.getconfigCtx.subDomainStatus.GetAll()
+		for _, dsItem := range dsItems {
+			ds := dsItem.(types.DomainStatus)
+			if ds.DomainName == aiStatus.DomainName {
+				ReportAppInfo.ClusterAppRunning = ds.Activated
```

Comment:
I think this entire commit should be just one line

ReportAppInfo.ClusterAppRunning = aiStatus.Activated

Reply:
yeah changed it to that